src/main/java/com/nvs/store/dto/cart/CartItemDto.java

```diff
@@ -0,0 +1,43 @@
+package com.nvs.store.dto.cart;
+
+import com.nvs.store.models.cart.Cart;
+import com.nvs.store.models.product.Product;
+
+public class CartItemDto {
+    private Integer id;
```

Comment:
If the id represents the unique identifier of table, recommend to exclude it because you can identify the item of cart based on cart id and product id.